pynucastro/networks/sympy_network_support.py

```diff
@@ -66,11 +69,21 @@ def ydot_term_symbol(self, rate, y_i):
         return result
 
     def specific_rate_symbol(self, rate):
-        """
-        return a sympy expression containing the term in a dY/dt equation
-        in a reaction network corresponding to this rate.
+        """Construct a SymPy expression containing this rate's term in
+        a dY/dt equation, e.g. ρ Y(A)Y(B) <σv> / (1 + ẟ_AB) for a
+        rate A + B
 
         Also enter the symbol and substitution in the lookup table.
+
+        Parameters
+        ----------
+        rate : Rate
+            the reaction rate to consider
+
+        Returns
+        -------
+        sympy.core.mul.Mul
```

Comment:
This return type (as well as for `jacobian_term_symbol`) would be better as `sympy.core.expr.Expr`

Reply:
updated